### ISSUE TITLE
Possible incompatibility with Relay 0.6.1

### ISSUE THREAD

**Issue by thelordoftheboards** [denvned/isomorphic-relay#11]
After upgrading to Relay 0.6.1 there seems to be a problem at https://github.com/denvned/isomorphic-relay/blob/master/src/prepareData.js

``` javascript
RelayQuery.Node.prototype.getConcreteQueryNode = onCacheMiss => onCacheMiss();
```

The error looks like:

```
TypeError: onCacheMiss is not a function
   at RelayQueryField._RelayQuery2.default.Node.getConcreteQueryNode (<project root>/node_modules/isomorphic-relay/lib/prepareData.js:55:12)
   at Object.<anonymous> (<project root>/node_modules/react-relay/lib/toGraphQL.js:99:19)
   at Object.Field (<project root>/node_modules/react-relay/lib/toGraphQL.js:87:21)
   at toGraphQLSelection (<project root>/node_modules/react-relay/lib/toGraphQL.js:121:22)
   at Array.map (native)
   at Object.Query (<project root>/node_modules/react-relay/lib/toGraphQL.js:48:39)
   at _class.handleQueryPayload (<project root>/node_modules/isomorphic-relay/lib/prepareData.js:78:60)
   at <project root>/node_modules/react-relay/lib/RelayPendingQueryTracker.js:221:25
   at <project root>/node_modules/react-relay/lib/RelayTaskQueue.js:118:25
   at <project root>/node_modules/react-relay/lib/RelayTaskQueue.js:124:15
```

**Comment by denvned** [comment]
I have just released isomorphic-relay 0.4.2 intended to be used with Relay 0.6.1.

**Comment by terrisgit** [comment]
Where is the +1 button?
